firmware/src/src/sense.cpp

```diff
@@ -433,13 +433,33 @@ void calculate_Thread()
         int tltch = trkset.tiltCh();
         int rllch = trkset.rollCh();
         int panch = trkset.panCh();
+        int alertch = trkset.alertCh();
         if(tltch > 0)
             channel_data[tltch - 1] = trpOutputEnabled == true ? tiltout_ui : trkset.Tlt_cnt();
         if(rllch > 0)
             channel_data[rllch - 1] = trpOutputEnabled == true ? rollout_ui : trkset.Rll_cnt();
         if(panch > 0)
             channel_data[panch - 1] = trpOutputEnabled == true ? panout_ui : trkset.Pan_cnt();
 
+        // TODO what is trpOutputEnabled?
```

Comment:
not sure if trpOutputEnabled will matter for alert pulse behavior

Reply:
No you shouldn't use it. It is there to disable the TRP outputs if user has selected the long press mode to enable/disable those outputs. Was a feature request to make it more like the Trinity head tracker

Reply:
ok -- then my change is probably orthogonal -- i'll remove the todo comment